include/seqan3/alignment/configuration/all.hpp

```diff
@@ -31,6 +31,12 @@
  * \brief A special sub namespace for the alignment configurations.
  */
 
+ /*!\cond DEV
+  * \namespace seqan3::align_cfg::detail
+  * \copydoc seqan3::detail
+  * \endcond
+  */
```

Comment:
```suggestion
 /*!\if DEV
  * \namespace seqan3::align_cfg::detail
  * \copydoc seqan3::detail
  * \endif
  */
```

We should use `if` for inside comment blocks and `cond` for exluding code / multiple comment blocks

Reply:
Is this the same?

I just copied the code from

https://github.com/seqan/seqan3/blob/aa45291d7aafd88d2d58c2e4fcf1702b1fbf6220/include/seqan3/core/all.hpp#L48-L56

Reply:
if would be better style; but then you can just leave it as it is.

Reply:
Changed it